scheduler/src/Rsbc.Dmf.Scheduler/Startup.cs

```diff
@@ -204,7 +204,7 @@ public void ConfigureServices(IServiceCollection services)
 
                 var httpClient = new HttpClient(httpClientHandler)
                 {
-                    Timeout = TimeSpan.FromMinutes(30)
+                    Timeout = TimeSpan.FromMinutes(60)
```

Comment:
Does this need to be larger?  Maybe we should make it a config setting in case it needs to be set in the field